src/app/locale/translations/en.json

```diff
@@ -100,7 +100,7 @@
         },
         "customer": {
             "checkout_as_guest_text": "Checking out as a <strong>Guest</strong>? You'll be able to save your details to create an account with us later.",
-            "continue_as_guest_action": "Continue as guest",
+            "continue_as_guest_action": "Continue",
```

Comment:
Why are we changing `Continue as guest` for everyone?

Reply:
Oh, I will fix it

Reply:
Fixed